CHANGES.rst

```diff
@@ -20,6 +20,12 @@ hsa
 Service fixes and enhancements
 ------------------------------
 
+hst
```

Comment:
Inner refactorings with unchanged user API arent' usually mentioned in the changelog, but if the speed improvement is significant we can leave this here.
```suggestion
esa.hubble
```